src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
@whipermr5 Either way, could the underlying issue be handled in a separate PR? Because changing the `alert` classes on ALL the modals in TEAMMATES seems like a huge job, and not one I'm sure I'm up for right now 😅